docs/installation.md

```diff
@@ -23,14 +23,14 @@ compile a runnable Mirror Node JAR file in the `target` directory.
 
 In addition to OpenJDK 11, you will need to install [PostgreSQL](https://postgresql.org) 9.6 and initialize it. The only
 setup required is to create the initial database and owner since [Flyway](https://flywaydb.org) manages the
-database schema. The SQL script located at `hedera-mirror-importer/src/main/resources/db/scripts/init.sql` can be used to
+database schema. The SQL script located at `hedera-mirror-importer/src/main/resources/db/scripts/0_init.sql` can be used to
```

Comment:
The SQL scripts located at `hedera-mirror-importer/src/main/resources/db/scripts/init` can be used to...